packages/protocol/lib/web3-utils.ts

```diff
@@ -227,51 +242,86 @@ export function deploymentForCoreContract<ContractInstance extends Truffle.Contr
   artifacts: any,
   name: CeloContractName,
   args: (networkName?: string) => Promise<any[]> = async () => [],
-  then?: (contract: ContractInstance, web3: Web3, networkName: string) => void
+  then?: (contract: ContractInstance, web3: Web3, networkName: string) => void,
+  artifactPath?: string
 ) {
-  return deploymentForContract(web3, artifacts, name, args, true, then);
+  return deploymentForContract(web3, artifacts, name, args, true, then, artifactPath);
 }
 
 export function deploymentForProxiedContract<ContractInstance extends Truffle.ContractInstance>(
   web3: Web3,
   artifacts: any,
   name: CeloContractName,
   args: (networkName?: string) => Promise<any[]> = async () => [],
-  then?: (contract: ContractInstance, web3: Web3, networkName: string) => void
+  then?: (contract: ContractInstance, web3: Web3, networkName: string, proxy?: ProxyInstance) => void,
+  artifactPath?: string
 ) {
-  return deploymentForContract(web3, artifacts, name, args, false, then);
+  return deploymentForContract(web3, artifacts, name, args, false, then, artifactPath);
 
 }
 
+
+export const makeTruffleContract = (contractName:string, contractPath:string, web3: Web3) => {
+  const artifact = require(`${path.join(__dirname, "..")}/build/contracts-${contractPath}/${contractName}.json`)
+  const Contract = truffleContract({
+    abi: artifact.abi,
+    unlinked_binary: artifact.bytecode,
+  })
+  
+  const {createInterfaceAdapter} = require("@truffle/interface-adapter")
+  
+  Contract.setProvider(web3.currentProvider)
+  Contract.setNetwork(1101)
+  Contract.interfaceAdapter = createInterfaceAdapter({networkType:"ethereum", provider:web3.currentProvider})
```

Comment:
Style nit: spaces after `:`. Maybe even separate object onto multiple lines for readability, like 

```suggestion
  Contract.interfaceAdapter = createInterfaceAdapter({
    networkType: "ethereum",
    provider: web3.currentProvider
  })
```